ext/atomic_ruby/atomic_ruby.c

```diff
@@ -41,9 +41,20 @@ static VALUE rb_cAtom_allocate(VALUE klass) {
   return obj;
 }
 
+#ifdef HAVE_RACTOR_SHAREABLE_PROC
+static void check_value_shareable(VALUE value) {
+  if (!rb_ractor_shareable_p(value)) {
```

Comment:
I was wondering since when this function is available but actually since the introduction of Ractor in 3.0: https://github.com/ruby/ruby/blob/ruby_3_0/include/ruby/ractor.h